server/src/main/java/org/opentosca/toscana/plugins/cloudfoundry/CloudFoundryFileCreator.java

```diff
@@ -77,7 +101,28 @@ private void createBuildpackAdditionsFile() throws IOException, JSONException {
         for (String buildPack : app.getBuildpackAdditions()) {
             buildPacks.put(buildPack);
         }
-        buildPackAdditionsJson.put("PHP-EXTENSIONS", buildPacks);
-        fileAccess.access("/.bp-config/options.json").append(buildPackAdditionsJson.toString()).close();
+        buildPackAdditionsJson.put(BUILDPACK_OBJECT_PHP, buildPacks);
+        fileAccess.access("/" + BUILDPACK_FILEPATH_PHP).append(buildPackAdditionsJson.toString()).close();
+    }
+
+    private void createAttributes() throws IOException {
+
+        if (app.getAttributes().size() > 0) {
+            ArrayList<String> attributes = new ArrayList<>();
+            for (Map.Entry<String, String> attribute : app.getAttributes().entrySet()) {
+                attributes.add("  " + attribute.getKey() + ": " + attribute.getValue());
```

Comment:
Could be replaced with:
`attributes.add(String.format("  %s: %s", attribute.getKey(), attribute.getValue()));`